docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+          "type": "number"
+        },
+        "educationTypes": {
+          "description": "Types of education",
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "tas-date": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-date",
+      "title": "tas-date",
+      "description": "A date item YYYY-MM-DD",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["date"]
+        },
+        "oldestDateAllowed": {
+          "description": "Oldest date that can be selected",
+          "type": "string"
+        },
+        "newestDateAllowed": {
+          "description": "Newest date that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-time": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-time",
+      "title": "tas-time",
+      "description": "A time item HH:MM:SS-HH:MM",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["time"]
+        },
+        "oldestTimeAllowed": {
+          "description": "Oldest time that can be selected",
+          "type": "string"
+        },
+        "newestTimeAllowed": {
+          "description": "Newest time that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-datetime": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-datetime",
+      "title": "tas-datetime",
+      "description": "A date item YYYY-MM-DDTHH:MM:SS-HH:MM",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["datetime"]
+        },
+        "oldestDateAllowed": {
+          "description": "Oldest datetime that can be selected",
+          "type": "string"
+        },
+        "newestDateAllowed": {
+          "description": "Newest datetime that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-salary": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-salary",
+      "title": "tas-salary",
+      "description": "A salary question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["salary"]
+        },
+        "payBands": {
+          "description": "List of options if freeform input not allowed",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "maximumDigitals": {
+          "description": "Max number characters",
+          "type": "number",
+          "default": 15
+        }
+      }
+    },
+
+    "tas-info": {
```

Comment:
This is for if you've got a piece of information to add in amongst the questions in the form, that doesn't require an answer.